prisma/schema.prisma

```diff
@@ -84,8 +69,16 @@ model ProductFeature {
 model Release {
   id                 Int                @id @default(autoincrement())
   name               String
```

Comment:
Nit: Presumably patch/bugfix-release numbers will be incorporated into the name. Do we want the patch version as a separate field? Might make some things easier to track (can't think of any concrete examples right now, just a hunch).